extra-dev/packages/coq-equations/coq-equations.dev+HoTT/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+authors: [ "Matthieu Sozeau <matthieu.sozeau@inria.fr>" "Cyprien Mangin <cyprien.mangin@m4x.org>" ]
+maintainer: "matthieu.sozeau@inria.fr"
+homepage: "https://mattam82.github.io/Coq-Equations"
+dev-repo: "git+https://github.com/mattam82/Coq-Equations.git"
+bug-reports: "https://github.com/mattam82/Coq-Equations/issues"
+license: "LGPL 2.1"
+build: [
+  ["./configure.sh HoTT"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/Equations"]
+depends: [
+  "ocaml"
+  "coq" {= "dev" }
+  "coq-hott" {= "dev" }
```

Comment:
Are you sure this should be a "version" of the `coq-equations` package as opposed to a package of its own (say, `coq-equations-hott`)? Note that as a version, it won't be possible to both have the regular `coq-equations.dev` version and the `coq-equations-dev+HoTT` version installed at the same time.